packages/semantic-ui/src/FieldTemplate/FieldTemplate.tsx

```diff
@@ -68,12 +68,12 @@ export default function FieldTemplate<
       <Form.Group key={id} widths='equal' grouped>
         <MaybeWrap wrap={wrapContent} className='sui-field-content'>
           {children}
-          {displayLabel && rawDescription && (
+          {displayLabel && description && (
             <MaybeWrap wrap={wrapLabel} className='sui-field-label'>
-              {rawDescription && (
+              {description && (
                 <DescriptionFieldTemplate
                   id={descriptionId<T>(id)}
-                  description={rawDescription}
+                  description={description}
```

Comment:
Hmmm, looking at the snapshots below it seems as if `description` already rendered as a `DescriptionFieldTemplate` since I'm seeing the following:
```
                      <p
                        className="sui-description"
                        id="root_1__description"
                      >
                        <p
                          className="sui-description"
                          id="root_1__description"
                        >
                          a test item description
                        </p>
                      </p>
```

In which case this simplifies to:
```
<MaybeWrap wrap={wrapLabel} className='sui-field-label'>{description}</MaybeWrap>
```

Reply:
Makes sense. I removed the `DescriptionFieldTemplate` component.